### ISSUE TITLE
Unable to make migrations for new model

### ISSUE THREAD

**Issue by Pratyush1991** [OpenDroneMap/WebODM#364]
### How did you install WebODM? (Docker, natively, ...)?
Docker

### What's your browser and operating system? (Copy/paste the output of https://www.whatismybrowser.com/)

Firefox 57 on windows 10

### What is the problem?

Unable to migrate new model in development environment

### What should be the expected behavior?

Need help in migrating models 

### How can we reproduce this? (What steps did you do to trigger the problem? What parameters are you using for processing? If possible please include a copy of your dataset uploaded on Google Drive or Dropbox. Be detailed)

I am currently running webodm in development environment. I added a new model in app but I am unable to migrate the model. Screenshot attached below.

![image](https://user-images.githubusercontent.com/28738918/34076024-248e3a8a-e30c-11e7-96f7-2e7225a98b46.png)

![image](https://user-images.githubusercontent.com/28738918/34076017-cf57372e-e30b-11e7-81b1-c8ed949c92a1.png)

I did run python manage.py makemigrations.
![image](https://user-images.githubusercontent.com/28738918/34076036-7dc5a50c-e30c-11e7-824b-a4b8c29d0adc.png)

**Comment by pierotofy** [comment]
You are running the makemigrations command from your host environment, which is not going to work (you need to run the command within the webodm_webapp docker container).

Or more simply, just recreate the container:

`./webodm.sh down && ./devenv.sh start`

**Comment by Pratyush1991** [comment]
Thanks a lot. I did try to recreate the container using above command still the changes were not reflected in the database not sure why. So I ran the migration command inside the docker container which solved my problem.

I would also like to know if I should post issues here or somewhere else for better communication as I am a bit new to development environment and often have some questions or queries.
